packages/dbs-ai/server/hooks/searchHooks.js

```diff
@@ -0,0 +1,16 @@
+/* global SystemLogger */
+function onSpotlighStartHandler(item) {
+	SystemLogger.debug('onSpotlightStart', item.text);
+}
+
+function onSpotlighEndHandler(item) {
```

Comment:
@lobo87 I expected the `accessibleRooms` to be a dedicated parameter. What's the reason you added it to `item`?

Reply:
or is "item" just the parameter created by RC as the `callbackParams`?

Reply:
Rocketchat callbakcs only receive one argument. this is why i added it to the item